baseapp/msg_service_router_test.go

```diff
@@ -22,7 +22,7 @@ import (
 
 func TestMsgService(t *testing.T) {
 	priv, _, _ := testdata.KeyTestPubAddr()
-	encCfg := simapp.MakeEncodingConfigTests()
+	encCfg := simapp.MakeEncodingConfig()
```

Comment:
Why do you change this?

Reply:
https://github.com/cosmos/cosmos-sdk/pull/7597/files#r510072036